CONTRIBUTING.md

```diff
@@ -1,94 +1,121 @@
 Contributing
 ============
 
-In this document: 
+Thank you for your interest in contributing to the .NET documentation!
+
+In this topic, you'll see the basic process for adding or updating content in our [documentation site](https://docs.microsoft.com/dotnet). For the detailed step-by-step process and instructions, please see the [official guide](https://github.com/Microsoft/Docs/blob/master/readme.md) in the [Microsoft/Docs](https://github.com/Microsoft/Docs) repo.
+
+In this topic, we'll cover: 
 
 * [Adding content](#adding-content)
 * [Process for contributing](#process-for-contributing) 
 * [Guidance checklist](#guidance-checklist)
 * [Building the docs](#building-the-docs)
 * [Contributing to samples](#contributing-to-samples)
+* [Contributor License Agreement](#contributor-license-agreement)
 
 ## Adding content ##
 
-Before adding content, please submit an issue describing your content proposal. For smaller changes or additions, feel free to make a pull request.
+Before adding content, please submit an issue describing your content proposal. For smaller changes or additions, feel free to make a Pull Request (PR) without opening an issue.
+
+The content inside the **docs** folder is organized into sections that are reflected in our Table of Contents (TOC). 
+See where your article fits in the TOC and navigate to the folder associated to that section.
+That folder contains the Markdown files for all articles in the section.
+If necessary, create a new folder to place the files for your content. The main article for that section will be the index.md. 
+For images and other static resources, create a subfolder called **media** inside the folder that contains your article, if it doesn't already exist. Inside the **media** folder, create a subfolder with the article name (except for the index file).
 
-Articles should be organized into logical groups or sections. Each section
-should be given a named folder (e.g. /yourfirst). That section contains the
-Markdown files for all articles in the section. For images and other static
-resources, create a subfolder that matches the name of the article. Within this
-subfolder, create a ``sample`` folder for code samples and a  ``_static`` folder
- for images and other static content.
+Larger samples should be included in the samples folder under the root of the repo. 
 
 ### Example structure ###
 
     docs
-      /concepts
-      /getting-started
-      /porting
-        /_static
-          portability_report.png
-	  ...
-
-Author information is kept in the Markdown file itself. Each author should have a link to an online presence for himself/herself. 
+      /about
+      /core
+        /porting
+          porting-overview.md
+          /media
+            /porting-overview        
+                portability_report.png
+      ...
+    samples
```

Comment:
I like that. I like that the recommendations represent where we want to go. I just want to avoid the critique that "you don't do what you want others to do." This is a great way to explain why.